pkg/apis/options/providers.go

```diff
@@ -56,9 +56,16 @@ type Provider struct {
 	// Name is the providers display name
 	// if set, it will be shown to the users in the login page.
 	Name string `json:"name,omitempty"`
+	// SSLInsecureSkipVerify disables verification of the providers CA
+	SSLInsecureSkipVerify bool `json:"insecureSkipVerify,omitempty"`
```

Comment:
I can do that. Any concerns with backwards compatibility? I think moving to a TLS object here instead would mean pulling in both `CAFiles` and `UseSystemTruststore`, so maybe the result here to avoid breaking things might look like using a json tag like `json:",inline"` so those fields don't get nested. Or did you have something else in mind?

Reply:
I'm happy about changing structures within the alpha config and breaking between Y (x.y.z) stream releases, so i'd prefer we get the config set up for the future as we think it should be, than worry about breaking something we are calling alpha at present